mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -521,13 +521,39 @@ func resourceContainerCluster() *schema.Resource {
 				Description: ` Description of the cluster.`,
 			},
 
-			"enable_binary_authorization": {
-				Default:     false,
```

Comment:
@ScottSuarez is removing the default considered a breaking change?

Reply:
yep, we shouldn't remove any defaults.